core/src/main/java/org/apache/iceberg/ClientPoolImpl.java

```diff
@@ -138,8 +137,7 @@ private void release(C client) {
     }
   }
 
-  @VisibleForTesting
-  int poolSize() {
+  public int poolSize() {
```

Comment:
Why make this public?

Reply:
@ismailsimsek: Is this public because previously the package was `org.apache.iceberg.hive` and now the files are moved to `org.apache.iceberg`, so package private access modifier is not enough anymore?

Reply:
@pvary correct, its not working when the test is in another module.